src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -537,6 +538,72 @@ void AppCommandlineArgs::_buildFocusPaneParser()
     setupSubcommand(_focusPaneShort);
 }
 
+void AppCommandlineArgs::_buildSaveParser()
+{
+    _saveCommand = _app.add_subcommand("x-save", RS_A(L"SaveActionDesc"));
+
+    auto setupSubcommand = [this](auto* subcommand) {
+        subcommand->add_option("--name,-n", _saveInputName, RS_A(L"SaveActionArgDesc"));
+        subcommand->add_option("--keychord,-k", _keyChordOption, RS_A(L"KeyChordArgDesc"));
+        subcommand->add_option("command,", _commandline, RS_A(L"CmdCommandArgDesc"));
+        subcommand->positionals_at_end(true);
+
+        // When ParseCommand is called, if this subcommand was provided, this
+        // callback function will be triggered on the same thread. We can be sure
+        // that `this` will still be safe - this function just lets us know this
+        // command was parsed.
+        subcommand->callback([&, this]() {
+            // Build the NewTab action from the values we've parsed on the commandline.
+            ActionAndArgs saveAction{};
+            saveAction.Action(ShortcutAction::SaveTask);
+            // _getNewTerminalArgs MUST be called before parsing any other options,

```

Comment:
a bit of copypasta here 😅

Reply:
Updated to be slightly more generic.  // Build the action from the values we've parsed on the commandline.

Does this make sense or should I just remove it?